### PR TITLE
Annotate markdown fields in API and database

### PR DESCRIPTION
**Pull request by kalvdans**
Patch that sprinkles "CommonMark without images" everywhere, feel free to thumbsdown if you think it pollutes the files rather than helping.

Not sure if references are to allowed to be Markdown formatted or just a string of words.

### CODE REVIEW

**Comment by lucaslcode** [comment]
Maybe this should wait until after #529 to avoid merge conflicts

File: app/backend/src/couchers/models.py [hunk 1]
```diff
@@ -534,7 +534,7 @@ class Message(Base):
     # time sent, timezone should always be UTC
     time = Column(DateTime(timezone=True), nullable=False, server_default=func.now())
 
-    # the message text if not control
+    # the CommonMark-compliant message text if not control
```
Comment (aapeliv) [inline app/backend/src/couchers/models.py]: I believe the group chat messages are plaintext. Is that right @lucaslcode?

File: app/backend/src/couchers/models.py [hunk 2]
```diff
@@ -394,7 +394,7 @@ class Reference(Base):
 
     reference_type = Column(Enum(ReferenceType), nullable=False)
 
-    text = Column(String, nullable=True)
+    text = Column(String, nullable=True)  # CommonMark without images
```
Comment (aapeliv) [inline app/backend/src/couchers/models.py]: The reference also is currently plaintext, I think

File: app/pb/api.proto [hunk 3]
```diff
@@ -299,7 +299,7 @@ message Reference {
   int64 from_user_id = 1;
   int64 to_user_id = 2;
   ReferenceType reference_type = 3;
-  string text = 4;
+  string text = 4;  // CommonMark without images
```
Comment (aapeliv) [inline app/pb/api.proto]: As above

File: app/pb/api.proto [hunk 4]
```diff
@@ -322,7 +322,7 @@ message AvailableWriteReferenceTypesRes {
 message WriteReferenceReq {
   int64 to_user_id = 1;
   ReferenceType reference_type = 2;
-  string text = 3;
+  string text = 3;  // CommonMark without images
```
Comment (aapeliv) [inline app/pb/api.proto]: As above

File: app/pb/conversations.proto [hunk 5]
```diff
@@ -109,7 +109,7 @@ message Message {
   uint64 author_user_id = 2;
   google.protobuf.Timestamp time = 3;
   oneof content {
-    MessageContentText text = 4;
+    MessageContentText text = 4;  // CommonMark without images
```
Comment (aapeliv) [inline app/pb/conversations.proto]: As above

**Review by aapeliv — CHANGES_REQUESTED** [review]

**Comment by aapeliv** [comment]
> Maybe this should wait until after #529 to avoid merge conflicts

#529 has now been merged.

**Comment by kalvdans** [comment]
> #529 has now been merged.

Thanks for the reminder! Merged latest develop and fixed your comments.

File: app/pb/conversations.proto (hunk 5 above)
Comment (aapeliv) [reply app/pb/conversations.proto]: I think this is still plain text

**Review by aapeliv — CHANGES_REQUESTED** [review]

**Review by aapeliv — APPROVED** [review]
LGTM!

**Pull request merged**